errand_matcher/models.py

```diff
@@ -41,5 +41,16 @@ class ConfirmationToken(models.Model):
     email = models.EmailField(default='livelyhood.tech@gmail.com')
 
 class Requestor(models.Model):
-    pass
+
+    CONTACT_PREFERENCE_CHOICES = (
+        (1, 'sms'),
+        (2, 'call')
+    )
+    user = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
+    first_name = models.CharField(max_length=30, default='')
```

Comment:
you don't need the first_name and last_name fields because they're already in the linked User model